lein-light-nrepl/src/lighttable/nrepl/eval.clj

```diff
@@ -70,7 +81,9 @@
      (and (string? res) (:verbatim opts)) res
      :else (pr-str res))))
 
-(defn truncate [v]
+(defn truncate
+  "Same as 'identity'. Deprecated and to be removed."
```

Comment:
As mentioned before, please remove. The function is self explanatory and the person who added has no knowledge of when this is to be deprecated and removed. Code todos should not be accepted from external contributors who have little to no relevant contributions